_maps/map_files/generic/CentCom_skyrat_z2.dmm

```diff
@@ -6253,12 +6308,39 @@
 	},
 /turf/open/floor/iron,
 /area/centcom/interlink)
+"enf" = (
+/obj/effect/turf_decal/tile/neutral{
+	dir = 1
+	},
+/obj/effect/turf_decal/tile/neutral,
+/obj/effect/turf_decal/tile/neutral{
+	dir = 4
+	},
+/obj/effect/turf_decal/tile/neutral{
+	dir = 8
+	},
+/obj/effect/turf_decal/trimline/blue/filled/line,
+/obj/machinery/vending/autodrobe{
+	default_price = 0;
+	extended_inventory = 1;
+	extra_price = 0;
+	fair_market_price = 0
+	},
```

Comment:
I don't actually think you need any of that, it loads at round-start and it's off-station, so it should be free.

Reply:
its a holdover from like, old days

Reply:
but removed anyway, yeah